packages/core-bridge/index.d.ts

```diff
@@ -111,7 +111,14 @@ export interface PrometheusMetricsExporter {
 /**
  * Metrics exporters supported by Core
  */
-export type MetricsExporter = PrometheusMetricsExporter | OtelCollectorExporter;
+export type MetricsExporter = {
+  /**
+   * Type of aggregation temporality for metric export.
+   *
+   * See the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/ce50e4634efcba8da445cc23523243cb893905cb/specification/metrics/datamodel.md#temporality) for more information.
+   */
+  temporality?: 'cumulative' | 'delta';
+} & (PrometheusMetricsExporter | OtelCollectorExporter);
```

Comment:
doesn't apply to prometheus, right? should we reflect that in type?

Reply:
It does apply to prometheus.

Reply:
Maybe also link to prom docs. The comment implies it'll be for just OTel